pytorch_tabnet/callbacks.py

```diff
@@ -0,0 +1,223 @@
+import time
+import datetime
+import copy
+import numpy as np
+from dataclasses import dataclass, field
+from typing import List, Any
+
+
+class Callback:
```

Comment:
Is this pytorch compatible? Can I use any scheduler for example and add it as a callback?

Reply:
You can use any scheduler using `scheduler_fn` and `scheduler_params`. Into the `fit` method you can add any custom callback using the dedicated parameter following the format define here

Reply:
I'll make an update to add a nice LR call back